dapr/proto/runtime/v1/dapr.proto

```diff
@@ -876,46 +877,50 @@ message SubtleVerifyAlpha1Response {
   bool valid = 1;
 }
 
-message WorkflowReference {
-  string instance_id = 1 [json_name = "instanceId"];
-}
-
 message GetWorkflowRequest {
-  string instance_id  = 1 [json_name = "instanceId"];
+  string instance_id  = 1 [json_name = "instanceID"];
   string workflow_component = 2 [json_name = "workflowComponent"];
-  string workflow_name = 3 [json_name = "workflowName"];
 }
 
 message GetWorkflowResponse {
-  string instance_id = 1;
-  int64 start_time = 2;
-  map<string, string> metadata = 3;
+  string instance_id = 1 [json_name = "instanceID"];
+  string workflow_name = 2 [json_name = "workflowName"];
+  google.protobuf.Timestamp start_time = 3 [json_name = "createdAt"];
```

Comment:
Does this specifically have to be timestamp? 
cc @dapr/maintainers-dapr

Reply:
Timestamp is the natural choice. Choosing `string` creates a maintainability problem since you rely on both sides of every timestamp field to agree on a particular format (e.g. RFC3339). It's also more efficient over the wire. I recommend the Dapr project use timestamps instead of strings in new protobuf contracts moving forward.

Reply:
I second @cgillum. If not `google.protobuf.Timestamp`, then perhaps a uint64 with a UNIX timestamp as another option.